ql.mm

```diff
@@ -10393,25 +9869,25 @@ Kalmbach axioms (soundness proofs) that require WOML
     TWDVFWDVTAUTVALOSNWEPVOWEPWDVFPWCVAPWCPAUGUHHHUINWFVOVFVBEZVGEZVGVOULVMWGVG
     VBVFMSWHVFVBVGEZEZVGVFVBVGLWJVFUTVABEZEZEVFVGEVGWIWLVFWLWIUTVABUJOQWLVGVFWK
     BUTVABVQRQQVFVGVRRTUMTTTUNVKVBVLVIABUOABUPUQABURUS $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 2, and j is set to 0.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 2, and j is set to 0.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i2j0 $p |- ( ( a ->2 b ) v ( a ->0 b ) ) = ( a ->0 b ) $=
     ( wn wa wo wi2 wi0 leor leao1 lel2or df-le2 df-i2 df-i0 2or 3tr1 ) BACZBCZD
     ZEZPBEZETABFZABGZEUBSTBTRBPHPQBIJKUASUBTABLABMZNUCO $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 2, and j is set to 1.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 2, and j is set to 1.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i2j1 $p |- ( ( a ->2 b ) v ( a ->1 b ) ) = ( a ->0 b ) $=
     ( wn wa wo wi2 wi1 wi0 leor leao1 lel2or lear lelor leo lerr ler lebi df-i2
     df-i1 2or df-i0 3tr1 ) BACZBCZDZEZUCABDZEZEZUCBEZABFZABGZEABHUIUJUFUJUHBUJU
     EBUCIUCUDBJKUGBUCABLMKUCUIBUCUHUFUCUGNOBUFUHBUENPKQUKUFULUHABRABSTABUAUB $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 2, and j is set to 4.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 2, and j is set to 4.  (Contributed by Roy F. Longton, 1-Jul-2005.)
```

Comment:
The Contributed line should be 3-Jul-2005 and there should be no Revised by. I realize that the original has both dates but there's no point in having two dates two days apart.